src/common/KokkosKernels_SimpleUtils.hpp

```diff
@@ -278,12 +278,9 @@ struct IsRelativelyIdenticalFunctor{
     typedef Kokkos::Details::ArithTraits<mag_type> KATM;
  
     mag_type val_diff = KAT::abs (view1(i) - view2(i));
-    mag_type denominator = KAT::abs(view1(i));
-    if(KAT::abs(view2(i)) > denominator)
-      denominator = KAT::abs(view2(i));
-    
-    if(denominator > KATM::zero() )
-      val_diff = val_diff / denominator;
+    if(KAT::abs(view1(i)) > KATM::zero() && KAT::abs(view2(i)) > KATM::zero()) {
```

Comment:
This looks fine, I am taking it that `view1` stores test values and `view2` stores reference values?
One small comment, it seems that `eps_type` should be the same as  `mag_type` for proper comparison.
Also technically you should not need to check `KAT::abs(view1(i)) > KATM::zero()` but it should not hurt to do so.